data/json/items/archery.json

```diff
@@ -840,7 +840,7 @@
       "price" : 324000, "//" : "Modern crossbows can come in anywhere from $300 to $2K.  Large plus Antique = expensive.",
       "material" : ["iron", "wood"],
       "flags" : ["FIRE_TWOHAND", "STR_RELOAD", "PRIMITIVE_RANGED_WEAPON", "TRADER_AVOID"],
-      "skill" : "archery",
+      "skill" : "rifle",
```

Comment:
Miniguns are rifles and not launchers and they're less similar to rifles than big xbows to small ones.